tests/import/comments.js

```diff
@@ -20,3 +20,12 @@ import {
   a as //comment3 //comment2 //comment1
   b
 } from "";
+
+import {
+  FN1,
```

Comment:
In the examples, can you add all the comment types: before same line, after same line, before own line, after own line?

I have a feeling that it's not going to work properly with this implementation